documentation/book/proc-metrics-deploying-prometheus.adoc

```diff
@@ -17,48 +17,44 @@ NOTE: By default, the Prometheus Operator only supports jobs that include an `en
 
 .Procedure
 
-. Download the Prometheus resource file from the repository and replace the example namespace with your own:
+. Modify the Prometheus installation file (`prometheus.yaml`) according to the namespace Prometheus is going to be installed in:
 +
 On Linux, use:
 +
 [source,shell,subs="+quotes,attributes"]
-curl -s https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
+sed -i 's/namespace: .*/namespace: _my-namespace_/' > prometheus.yaml
```

Comment:
Sorry I did not mention it before, but please change all the `sed` commands in this PR.

Reply:
Sorry. Missed the other two